subsys/spm/spm.c

```diff
@@ -334,6 +334,8 @@ static void spm_config_peripherals(void)
 		PERIPH("NRF_TIMER1", NRF_TIMER1_S, CONFIG_SPM_NRF_TIMER1_NS),
 		PERIPH("NRF_TIMER2", NRF_TIMER2_S, CONFIG_SPM_NRF_TIMER2_NS),
 		PERIPH("NRF_SAADC", NRF_SAADC_S, CONFIG_SPM_NRF_SAADC_NS),
+		PERIPH("NRF_PWM0", NRF_PWM0_S, CONFIG_SPM_NRF_PWM0_NS),
+		PERIPH("NRF_PWM1", NRF_PWM1_S, CONFIG_SPM_NRF_PWM1_NS),
```

Comment:
Maybe we should add the rest of the PWM's as well? While we are at it.

Reply:
Done